config/tra.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/*
+ * Copyright (c) 2024. Wepesi Dev Framework
+ */
+/**
+ * needs a description
+ * @param string $content
+ * @return mixed|string
+ */function tr($content)
+{
+    $args = func_get_args();
+    return tra($content, '', false, array_slice($args, 1));
+}
+
+/**
+ * translate your text
+ * @param string $message
+ * @param string|array    $value
+ * @return string
+ */
+function tra(string $message, $value = null): string
+{
+    $i18n = new \Wepesi\Core\i18n(\Wepesi\Core\Session::get('lang'));
```

Comment:
This approach of loading the default language from session on not the right way. we should have another method to check the default language from the `.env` file, and then, set it up on the session.

Reply:
Updated